website/docs/cloud-docs/agents/metrics.mdx

```diff
@@ -152,3 +152,5 @@ section are prefixed by `tfc-agent.core.policy.`.
 | `setup_policies.milliseconds`                  | Timer | Time spent setting up individually managed policies.     |
 | `setup_policy_engines.milliseconds`            | Timer | Time spent setting up policy runtimes.                   |
 | `setup_subjects.milliseconds`                  | Timer | Time spent setting up subjects for policy enforcement.   |
+| `write-sentinel-config.milliseconds`           | Timer | Time spent generating the Sentinel config file.          |
```

Comment:
It looks like the previous metrics are using snake case and these additions use kebab. Just wanted to double check that is accurate.

Reply:
thanks for catching that! fixed in the agent 👍